buildSrc/src/main/kotlin/profiler.publication.gradle.kts

```diff
@@ -0,0 +1,49 @@
+import java.net.URI
+
+plugins {
+    id("maven-publish")
+}
+
+publishing {
+    publications {
+        register<MavenPublication>("mavenJava") {
+            from(components["java"])
+
+            if (project.parent == null) {
+                afterEvaluate {
+                    val profilerDistribution = artifacts.add("archives", tasks.named<Zip>("distZip").flatMap { it.archiveFile }) {
```

Comment:
This should go into the root project. I don't see why the general publication plugin would add the zip.